### PR TITLE
Chore/e2e test unregistered user

### PR DESCRIPTION
**Pull request by EmiM**
### Pull Request Checklist

- [x] I have linked this PR to related GitHub issue.
- [x] I have updated the CHANGELOG.md file with relevant changes (the file is located at the root of monorepo).

### CODE REVIEW

**Review by Kacper-RF — APPROVED** [review]

File: packages/e2e-tests/src/selectors.ts [hunk 1]
```diff
@@ -256,6 +272,32 @@ export class Channel {
...
+      const labels = await this.driver.findElements(By.xpath(`//*[contains(@data-testid, "userLabel-${username}")]`))
+      const properLabels = labels.filter(async labelElement => {
+        const labelText = await labelElement.getText()
+        return labelText === label
+      })
+      return properLabels.length > 0
+    })
+  }
+
+  async waitForLabelsNotPresent(username: string) {
```
Comment (leblowl) [inline packages/e2e-tests/src/selectors.ts:293]: Are these two functions being used anywhere?

**Review by leblowl — COMMENTED** [review]
idea: perhaps squash commits to keep git history cleaner on develop

File: packages/e2e-tests/src/selectors.ts (hunk 1 above)
Comment (EmiM) [reply packages/e2e-tests/src/selectors.ts:293]: waitForLabel* - yes, in multipleClients.test.ts
getUserMessagesWrapper - no, that one I can delete.

Comment (leblowl) [reply packages/e2e-tests/src/selectors.ts:293]: Ok, if it's not being used, I'd suggest removing it unless it's going to be used soon in a subsequent PR.

File: packages/state-manager/src/sagas/users/users.selectors.ts [hunk 2]
```diff
@@ -82,6 +82,10 @@ export const allUsers = createSelector(csrsMapping, certificatesMapping, (csrs,
     const username = csrs[pubKey].username
     const isDuplicated = duplicateUsernames.includes(username)
     const isRegistered = Boolean(certs[pubKey])
+    console.log('isRegistered', isRegistered)
+    console.log('certs[pubKey]', certs[pubKey])
+    console.log('pubKey', pubKey)
+    console.log('-----')
```
Comment (leblowl) [inline packages/state-manager/src/sagas/users/users.selectors.ts:88]: Are these logs just for testing?

**Pull request merged**